src/JanusFtl.cpp

```diff
@@ -215,7 +215,7 @@ void JanusFtl::IncomingRtcp(janus_plugin_session* handle, janus_plugin_rtcp* pac
             handlePsfbRtcpPacket(handle, rtcpHeader);
             break;
         default:
-            spdlog::info("Got unknown RTCP packet! Type: {}", rtcpHeader->type);
+            spdlog::info("Got unknown RTCP packet! Type: {}", (uint16_t)rtcpHeader->type);
```

Comment:
Prefer `static_cast`

Reply:
TIL static_cast has better compile time checks than C-style casts, thanks for the catch.